### PR TITLE
[#4] Fix: Some typos identified and could be fixed

### PR DESCRIPTION
**Pull request by malparty**
https://github.com/nimblehq/git-template/issues/4

## What happened 👀

Fixed some typos
Use American English for `behavior` (previously `behaviour`).
 
## Insight 📝

`N/A`
 
## Proof Of Work 📹

`N/A`

### CODE REVIEW

**Review by carryall — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -2,12 +2,12 @@ https://github.com/nimblehq/git-template/issues/??
...
 ## Insight 📝
 
-Describe in details how to test the changes, which solution you tried but did not go with, referenced documentation is welcome as well.
- 
+Describe in detail how to test the changes, which solution you tried but did not go with, referenced documentation is welcome as well.
+
 ## Proof Of Work 📹
 
-Show us the implementation: screenshots, gif, etc.
+Show us the implementation: screenshots, GIT, etc.
```
Comment (doannimble) [inline .github/PULL_REQUEST_TEMPLATE.md]: @malparty Should it be `GIF`?  😄

Comment (malparty) [reply .github/PULL_REQUEST_TEMPLATE.md]: Nice catch! >< 
10ead89 

I've also pluralize GIFs as suggested by Thuumb in the `git-templates` repository~

**Review by rosle — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/chore_template.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
+about: "Open a chore issue for a minor update."
 title: "Update "
 labels: "type : chore"
 ---
 
 ## Why
 
-Describe the update details and why it's needed. 
- 
+Describe the update details and why it's needed.
```
Comment (hoangmirs) [inline .github/ISSUE_TEMPLATE/chore_template.md]: Should we change to
```suggestion
Describe the update in detail and why it is needed.
```

the same as you did in https://github.com/nimblehq/git-template/pull/20/files#diff-2c69b215001999ab79729762d7a6f0567d9dc71323961a98641c57be26bbb42fR10

**Review by hoangmirs — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/chore_template.md (hunk 2 above)
Comment (malparty) [reply .github/ISSUE_TEMPLATE/chore_template.md]: d880b24 ✅

**Review by doannimble — APPROVED** [review]
LGTM 💯

**Review by junan — APPROVED** [review]

**Comment by junan** [comment]
@malparty Can we update these changes in our [git template](https://github.com/nimblehq/git-template/tree/main/.github/ISSUE_TEMPLATE)  issue templates? We can refer to this Git template issues  content as a single source of truth and then update other projects later so all GitHub issue templates to keep the content same every repo. What do you think?

**Comment by malparty** [comment]
> e update these changes in our [git template](https://github.com/nimblehq/git-template/tree/main/.github/ISSUE_TEMPLATE) issue templates

At the moment I did a manual fix in https://github.com/nimblehq/git-template/pull/20 
But in the long term, whenever this repo change, it could be interesting to run a quick set of many COPY/PASTE PR... I'm wondering if this is something we can automate maybe... 🤔

**Review by luongvo — APPROVED** [review]

**Pull request merged**